protoc-gen-grpc-gateway/gengateway/template_test.go

```diff
@@ -391,8 +391,8 @@ func TestApplyTemplateRequestWithClientStreaming(t *testing.T) {
 		if want := spec.sigWant; !strings.Contains(got, want) {
 			t.Errorf("applyTemplate(%#v) = %s; want to contain %s", file, got, want)
 		}
-		if want := `marshaler.NewDecoder(newReader()`; !strings.Contains(got, want) {
-			t.Errorf("applyTemplate(%#v) = %s; want to contain %s", file, got, want)
+		if want := `marshaler.NewDecoder(newReader()`; strings.Contains(got, want) {
+			t.Errorf("applyTemplate(%#v) = %s; should not contain %s", file, got, want)
```

Comment:
This test is a bit silly, so just remove this if case.

Reply:
removed.